sabnzbd/utils/certgen.py

```diff
@@ -12,33 +12,24 @@
 from cryptography.x509.oid import NameOID
 import datetime
 import os
-import struct
+from sabnzbd.getipaddress import localipv4, ipv6
```

Comment:
the main scenario is sab kicking off the cert gen process..  tried to keep the code as close to the tutorial as possible (which is what we had before).. and the changes we made to the file locally I just cleaned up so it covers the scenario if the user wanted to run it manually and less likely to get different results